articles/TOC.yml

```diff
@@ -46,15 +44,23 @@
     href: language/toc.yml
   - name: Quantum simulators and resource estimators
     href: machines/toc.yml
-- name: Q# libraries
-  href: libraries/toc.yml
-- name: Quantum computing concepts
-  href: concepts/toc.yml
-- name: Further reading
-  href: for-more-info.md
-- name: Contributing to the QDK
-  href: contributing/toc.yml
-  topicHref: contributing/index.md
-- name: QDK Release Notes
-  topicHref: relnotes/index.md
-  href: relnotes/toc.yml
+- name: Concepts 
+  items: 
+  - name: Quantum computing concepts
+    href: concepts/toc.yml
+  - name: Q# libraries
+    href: libraries/toc.yml
+- name: Reference 
```

Comment:
It looks like there's now an empty Reference section as well as the "API Reference" section from the other docset; can we revert and fix before merging back to master? Thanks!

![image](https://user-images.githubusercontent.com/31516/80990623-76e87c00-8deb-11ea-9dcf-8251d8b167ca.png)

Reply:
+@Bradben @cjgronlund